packages/dcos-net/extra/dcos-net-setup.py

```diff
@@ -52,6 +52,7 @@ def main():
 
 def add_networkd_config(src):
     networkd = b'systemd-networkd.service'
+    networkd_path = '/etc/systemd/network'
```

Comment:
is there a reason the `networkd` string is annotated with the `b` and the `networkd_path` is not?

Reply:
`subprocess.run()` returns binary data and `b''` is here just for networkd to avoid type casting.

Reply:
The type of `result.stdout` is a`bytes` so it needs to be a byte string to do `if networkd not in result.stdout:`. The alternative would be to do `if networkd not in result.stdout.decode():` to compare it as utf-8 strings instead of opaque bytes.